charts/tfy-notebook-controller/README.md

```diff
@@ -4,7 +4,61 @@ This Helm chart package, provided by TrueFoundry, contains configurations and re
 
 ## Parameters for tfy-notebook-controller
 
-############## Parameters for tfy-notebook-controller
+############### Parameters for tfy-notebook-controller
```

Comment:
There should not be any changes in this file.

Reply:
this was autogenerated by github actions